tests/FSharp.Compiler.UnitTests/FSharp.Compiler.UnitTests.fsproj

```diff
@@ -16,11 +16,6 @@
     <Compile Include="HashIfExpression.fs" />
     <Compile Include="ProductVersion.fs" />
     <Compile Include="EditDistance.fs" />
-    <Compile Include="Compiler.fs" />
```

Comment:
Did you mean to remove these, or is this temporary? Thanks!

Reply:
Yes, I believe so. I moved these tests to FSharp.Tests.FSharpSuite instead.